transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/VertxRestDispatcher.java

```diff
@@ -84,7 +84,7 @@ public void init(Router router) {
   }
 
   protected void failureHandler(RoutingContext context) {
-    LOGGER.error("http server failed.", context.failure());
+    LOGGER.error("http server failed.", context.failure() == null ? null : context.failure().getCause());
```

Comment:
Logger is fine. See line 154, do not send the internal message to client.

Reply:
done